source/tutorials/nvidia.rst

```diff
@@ -320,6 +393,41 @@ Troubleshooting
   installation files into a directory named
   :file:`NVIDIA-Linux-x86_64-<VERSION>`.
 
+* The X server logs under :file:`/var/log/X*` contain useful
+  information about display and driver loading. Check all the files and
+  timestamps when troubleshooting.
+
+* The DKMS build logs under :file:`/var/lib/dkms/nvidia*` contain information
+  about kernel module builds which can be useful if the NVIDIA driver breaks
+  between kernel upgrades.
+
+
+No display or blank screen
+==========================
+
+Check if the has come up on another graphics device, including the integrated
+graphics, if you see a black screen or the login screen has not come up after
+installing the NVIDIA drivers. This can happen before an Xorg config has been
```

Comment:
The first sentence is really confusing.

Reply:
Yeah, yikes. Does my second attempt read better?